docs/home/contribute/participating.md

```diff
@@ -0,0 +1,97 @@
+
+Our automation assigns reviewers to pull requests, and contributors can request your review with a comment on the pull request: `/assign [your_github_handle]`. To indicate that a pull request requires no further changes, a reviewer adds a comment to the pull request `/lgtm`.
+
+Approvers can indicate their approval with a comment: `/approve`.
+
+Reviewers and approvers can add a `/hold` comment to prevent the pull request from being merged. Another reviewer or approver can remove a hold with the comment: `/hold cancel`.
+
+When both of a `lgtm` and `approval` tag have been added, and there are no `hold` tags, the automation on the repository will merge the pull request.
+
+**Note:** There is a special case when an approver uses the comment: `/lgtm`. In these cases, the automation will add both `lgtm` and `approve` tags, skipping any further review.
+{: .note }
+ 
+When a reviewer is assigned a pull request to review it is not a sole responsibility, and any other reviewer may also offer their opinions on the pull request. 
+If a reviewer is requested, it is generally expected that the PR will be left to that reviewer to do their editorial pass on the content. 
+For small and/or trivial pull requests, the assigned reviewer may be skipped if an approver thinks the content is sufficient and well understood. 
+
+- Approvers
+
+Approvers have the ability to merge a PR. 
+An approver can indicate `/lgtm` or `/approve` in a PR comment to have a pull request merged, and all pull requests require at least one approver to provide their vote in order for the PR to be merged.
+
+For PRs that require no review (typos or otherwise trivial changes), approvers can enter an `lgtm` comment, indicating no need for further review and flagging the PR with approval to merge.
+
+
+### Teams and groups within SIG-DOCS
+
+You can get an overview of [SIG-DOCS from the community github repo](https://github.com/kubernetes/community/tree/master/sig-docs). 
+The SIG Docs group defines two teams on Github:
+ - [@kubernetes/sig-docs-maintainers](https://github.com/orgs/kubernetes/teams/sig-docs-maintainers)
+ - [@kubernetes/sig-docs-pr-reviews](https://github.com/orgs/kubernetes/teams/sig-docs-pr-reviews)
+
+These groups maintain the [Kubernetes website repository](https://github.com/kubernetes/website), which houses the content hosted at this site.
+Both can be referenced with their `@name` in github comments to communicate with everyone in that group.
+
+These groups overlap, but aren't identical, to the automation tooling that Kubernetes uses as a project. For assignment of issues, pull requests, and to support PR approvals, the automation uses an OWNERS file for the repository. The approvers and reviewers for PR assignment and automation tooling work from the OWNERS file. 
```

Comment:
Remove ", but aren't identical,", and you have "These groups overlap to the automation tooling". Maybe this: These groups overlap, but do not exactly match, the groups used by the automation tooling ...